app/models/game.rb

```diff
@@ -3,12 +3,18 @@ class Game < ActiveRecord::Base
   belongs_to :black_player, class_name: 'User'
   belongs_to :winner, class_name: 'User'
   has_many :pieces
-  after_create :populate_board!
+  after_create :populate_board!, :set_captured_to_false
 
   def open_seat?
     white_player_id.nil? || black_player_id.nil?
   end
 
+  def set_captured_to_false
+    pieces.each do |piece|
+      piece.update_attributes(captured: false)
+    end
+  end
```

Comment:
I have a cool trick for this. :)

Instead of doing this in Ruby. We can have the database do it for us.

You can set a default value in Postgres. 

Create a migration and add the following...

```
change_column_default(:pieces, :captured, false)
```

More info: http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_default